st/clitests/awss3api_spec.py

```diff
@@ -2022,6 +2022,43 @@ def get_md5_sum(fname):
 AwsTest('Aws can delete bucket').delete_bucket("target-bucket")\
     .execute_test().command_is_successful()
 
+#******** Create Bucket ********
+AwsTest('Aws can create bucket').create_bucket("seagatebucket").execute_test().command_is_successful()
+
+#******** Get Bucket Versioning default status ********
+AwsTest('Aws can get bucket versioning default status').get_bucket_versioning("seagatebucket").execute_test().command_response_should_be_empty()
+
+#******** Enable Versioning on Bucket ********
+AwsTest('Aws can enable versioning on bucket').put_bucket_versioning("seagatebucket", "Enabled").execute_test().command_is_successful()
+
+#******** Get Bucket Versioning Enabled status ********
+AwsTest('Aws can get bucket versioning Enabled status').get_bucket_versioning("seagatebucket").execute_test().command_response_should_have("Enabled")
+
+# TODO: uncomment in specific version delete as empty bucket not possible without this api.
+#******** Put Object in Bucket Versioning Suspended status ********
+#AwsTest('Aws can get bucket versioning Suspended status').get_bucket_versioning("seagatebucket").execute_test().command_response_should_have("Suspended")
+
+#******** Delete object in enable versioning state ********
+#AwsTest('Aws can delete object in versioning state').delete_object("seagatebucket", "simpleobj1").execute_test().command_is_successful()
```

Comment:
Please don't add disabled tests in new code. Just implement them when needed.

Reply:
Removed Code